supplements/extra-life/adventure-with-muk.xml

```diff
@@ -0,0 +1,182 @@
+			<stat name="wisdom" value="1" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +2 (Dankwood Goblin)" supports="Custom Ability Score Increase 2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +1 (Dankwood Goblin)" supports="Custom Ability Score Increase 1" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<stat name="innate speed" value="30" bonus="base"/>
+			<grant type="Size" id="ID_SIZE_SMALL" />
+			<grant type="Vision" id="ID_VISION_DARKVISION" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Language" id="ID_LANGUAGE_GOBLIN" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Racial Trait" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_SPEAK_WITH_SMALL_BEASTS" />
+			<grant type="Racial Trait" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_NIMBLE_ESCAPE" />
+		</rules>
+	</element>
+	<element name="Speak with Small Beasts" type="Racial Trait" source="Adventure with Muk" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_SPEAK_WITH_SMALL_BEASTS">
+		<description>
+			<p>Through sounds and gestures, you can communicate simple ideas with Small or smaller beasts. Dankwood goblins love animals and often keep squirrels, badgers, rabbits, moles, woodpeckers, and other creatures as beloved pets.</p>
+		</description>
+		<sheet>
+			<description>Through sounds and gestures, you can communicate simple ideas with Small or smaller beasts.</description>
+		</sheet>
+	</element>
+	<element name="Nimble Escape" type="Racial Trait" source="Adventure with Muk" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_NIMBLE_ESCAPE">
+		<description>
+			<p>You can take the Disengage or Hide action as a bonus action on each of your turns.</p>
+		</description>
+		<sheet>
+			<description>You can take the Disengage or Hide action as a bonus action on each of your turns.</description>
+		</sheet>
+	</element>
+	<element name="Birdsquirrel" type="Companion" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_BIRDSQUIRREL">
+		<supports>Familiar, Variant Familiar</supports>
+		<description />
+		<setters>
+			<set name="strength">6</set>
+			<set name="dexterity">14</set>
+			<set name="constitution">13</set>
+			<set name="intelligence">4</set>
+			<set name="wisdom">12</set>
+			<set name="charisma">6</set>
+			<set name="ac">13</set>
+			<set name="hp">10</set>
+			<set name="speed">20 ft., climb 20 ft., fly 30 ft.</set>
+			<set name="languages">—</set>
+			<set name="type">Beast</set>
+			<set name="size">Tiny</set>
+			<set name="alignment">unaligned</set>
+			<set name="challenge">1/4</set>
```

Comment:
so it's not really standard yet (as i've didn't go back and didn't change it in every file), but we've started to change setters to reflect actual way they're displayed in statblocks and in spells, again, for readability sake. So i will modify these setters to reflect typical statblock too.
```suggestion
			<set name="size">Tiny</set>
			<set name="type">Beast</set>
			<set name="alignment">unaligned</set>
			<set name="ac">13</set>
			<set name="hp">10</set>
			<set name="speed">20 ft., climb 20 ft., fly 30 ft.</set>
			<set name="strength">6</set>
			<set name="dexterity">14</set>
			<set name="constitution">13</set>
			<set name="intelligence">4</set>
			<set name="wisdom">12</set>
			<set name="charisma">6</set>
			<set name="languages">—</set>
			<set name="challenge">1/4</set>
```